package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@babel/runtime": "7.7.4",
     "@babel/runtime-corejs3": "7.7.4",
+    "crypto-js": "^3.1.9-1",
```

Comment:
Can you pin the package and resolve the conflicts? Pinning means remove the ^

Reply:
Done 👌